### ISSUE TITLE
rspec error on documentation example

### ISSUE THREAD

**Issue by ygamretuta** [rebelidealist/stripe-ruby-mock#235]
given this example on docs:

```
RSpec.configure do |c|
  if c.filter_manager.inclusions.keys.include?(:live)
    puts "Running **live** tests against Stripe..."
    StripeMock.toggle_live(true)
  end
end
```

I get:

```
undefined method `keys' for #<RSpec::Core::InclusionRules:0x007fd419dc0290> (NoMethodError)
```

any ideas?

my rspec version (from `bundle list`) is: 

```
*rspec (3.2.0)
```

**Comment by alexmamonchik** [comment]
I can't reproduce that. How you did it?

**Comment by alexmamonchik** [comment]
found. Use `rules` instead of `keys`

**Comment by mindeavor** [comment]
@ygamretuta The docs state RSpec (2.x); you're using 3.2

**Comment by kathyonu** [comment]
Eye send my Thank You to @ygmaretuta for asking the original question ..
Thank You to @alexmamonchik for the, rules not keys, answer.
I gave up trying to run live tests a long time ago .. and when I saw this answer,
we now instantly have live tests working.

For the record, and to be crystal clear, this works to run Live Tests, and we are using RSpec 3.2 :

/spec/rails_helper.rb
-------------------------
RSpec.configure do |config|
  if config.filter_manager.inclusions.rules.include?(:live)
    StripeMock.toggle_live(true)
    Rails.logger.info(“*** Now running LIVE TESTS against Stripe ***")
  end
end

when run with this command : 

$	rspec -t live


For the sake of all please change this page : 

https://github.com/rebelidealist/stripe-ruby-mock
	
to this new and true code : 
----------------------------------
Here is an example of setting up your RSpec (2.x and later) test suite to run live with a command line switch:

  if config.filter_manager.inclusions.rules.include?(:live)
    StripeMock.toggle_live(true)
    Rails.logger.info(“*** Now running LIVE TESTS against Stripe ***")
  end


from this old and not working code : 
----------------------------------------------

Here is an example of setting up your RSpec (2.x) test suite to run live with a command line switch:
RSpec.configure do |c|
  if c.filter_manager.inclusions.keys.include?(:live)
    puts "Running **live** tests against Stripe..."
    StripeMock.toggle_live(true)
  end
end

**Comment by mindeavor** [comment]
Closed by #239
